GENERATE.md

```diff
@@ -1,3 +1,4 @@
+### OUT OF DATE ###
```

Comment:
Then remove the file?

Reply:
it still contains relevant information for website generation, but updating it does not make much sense. This will be removed once we stop using kw-doc